### PR TITLE
Add SEO component

### PR DESCRIPTION
**Pull request by micorix**

### CODE REVIEW

**Review by mlazowik — APPROVED** [review]

File: src/components/SEO.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    url: `${siteUrl}${pathname}`,
+  };
+
+  return (
+    <Helmet title={seo.title} titleTemplate={titleTemplate}>
+      <meta name="description" content={seo.description} />
+      {seo.url && <meta property="og:url" content={seo.url} />}
+      {(isArticle ? true : null) && <meta property="og:type" content="article" />}
+      {seo.title && <meta property="og:title" content={seo.title} />}
+      {seo.description && <meta property="og:description" content={seo.description} />}
```
Comment (MariaRukat) [inline src/components/SEO.tsx:33]: Is it necessary to check if we have seo.description or seo.title? We have default values for it so I guess we don't need to check it.

File: src/components/SEO.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      {twitterUsername && <meta name="twitter:creator" content={twitterUsername} />}
```
Comment (MariaRukat) [inline src/components/SEO.tsx:34]: Same here - 'twitterUsername' is defined.

**Review by MariaRukat — APPROVED** [review]

File: src/components/SEO.tsx (hunk 1 above)
Comment (micorix) [reply src/components/SEO.tsx:33]: Yeah, right

**Comment by micorix** [comment]
Also, please do not merge yet. We have to setup env variables.

**Comment by mlazowik** [comment]
@micorix what env vars?

**Comment by micorix** [comment]
nvm. I thought it would be good to set `SITE_URL` env var. for hostname. I'll just get it from window.location.

**Review by MariaRukat — APPROVED** [review]

**Review by MariaRukat — APPROVED** [review]

**Pull request merged**